### PR TITLE
[API-10782] About page and global nav updates

### PR DESCRIPTION
**Pull request by Cgfor3**
### Description

Related Tickets:
- https://vajira.max.gov/browse/API-10813
- https://vajira.max.gov/browse/API-10782
- https://vajira.max.gov/browse/API-10809

I added the About page to the global navigation and created a new page. The News page has been moved under the About page section.

### Process

- [x] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [x] Accessibility concerns have been considered and addressed
  
  - [x] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

**Review by imcgonigle — COMMENTED** [review]
This looks good to me. I'll add the approval once the build passes.

It seems to me that the Code scanning error can be dismissed since all of the URLs are coming from a file within the app and not from an external source. Alternatively you could change that to a regex match or create a URL object and grab the host.
```javascript
let url = new URL('https://www.youtube.com/watch?v=jmnH42Qse48');
if(url.host === 'www.youtube.com'); // true
```

**Comment by SlakrHakr** [comment]
Agree with @imcgonigle above. Let me know if i can help getting the last couple of pieces of the build passing in any way.

File: src/containers/about/About.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { ContentWithNav, SideNavEntry } from '../../components';
+// import { PAGE_HEADER_AND_HALO_ID } from '../../types/constants';
```
Comment (SlakrHakr) [inline src/containers/about/About.tsx:4]: nit: probably worth taking out

**Review by SlakrHakr — APPROVED** [review]

File: src/containers/about/Overview.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        'vads-u-padding-bottom--1',
+        'vads-u-margin-top--5',
+      )}
+    >
+      There&apos;s more to explore
+    </h3>
+    <ul>
+      <li>
+        Learn about{' '}
+        <a href="https://developer.va.gov/onboarding/working-with-lighthouse-apis">
```
Comment (gileswells) [inline src/containers/about/Overview.tsx:108]: This should use `<Link to="/onboarding/working-with-lighthouse-apis">` or better yet `<Link to={CONSUMER_APIS_PATH}>`

File: src/containers/about/Overview.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          working with our APIs
+        </a>
+        .
+      </li>
+      <li>
+        Access to our sandbox environment is automatic when you{' '}
+        <a href="https://developer.va.gov/apply">request an API key</a>.
```
Comment (gileswells) [inline src/containers/about/Overview.tsx:115]: Similar to above. `<Link to="/apply">`

File: src/containers/about/Overview.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        <a href="https://www.oit.va.gov/services/trm/">
+          Office of Information and Technology (OIT) standards and technology
+        </a>{' '}
+        to provide these approved entities—our consumers— with the highest quality integration
+        experience.
+      </p>
+      <ParagraphHeading>What do we do?</ParagraphHeading>
+      <p className="vads-u-margin-top--0">
+        We give our consumers no-cost access to VA data through{' '}
+        <a href="https://developer.va.gov/explore">our APIs</a>. We never, ever charge fees.
```
Comment (gileswells) [inline src/containers/about/Overview.tsx:35]: Should use `<Link to="/explore">`

**Review by gileswells — CHANGES_REQUESTED** [review]
A few small link changes to allow them to single page app link across without having to reload the whole app.

**Comment by Eallan919** [comment]
🥰

**Review by gileswells — APPROVED** [review]
Changes looking good to me.

**Pull request merged**